CHANGELOG.rst

```diff
@@ -1,5 +1,33 @@
 Changelog
 =========
+1.23.0 (2022-01-17)
+------------------------
+MLflow 1.23.0 includes several major features and improvements:
+
+Features:
+
+- [Models] Introduce an ``mlflow.evaluate()`` API for evaluating MLflow Models, providing performance and explainability insights. For an overview, see https://mlflow.org/docs/latest/models.html#model-evaluation (#5069, #5092, #5256, @WeichenXu123)
```

Comment:
This link will become valid when https://github.com/mlflow/mlflow/pull/5272 is merged and docs are published. Good for publicity to include link in the release notes.